script/common/check_xcode_version

```diff
@@ -1,13 +1,24 @@
-check_xcode_version ()
-{
-  if [ -z "$DISABLE_XCODE_CHECK" ]
-  then
-    installed_xcode_version=`xcodebuild -version | head -n 1 | ruby -e 'puts STDIN.read.match(/Xcode ((\d+\.)?\d+\.\d+)/)[1]'`
-    if [ "$installed_xcode_version" != "$REQUIRED_XCODE_VERSION" ]
-    then
-      printf "\033[1;31mError: xcodebuild version '$installed_xcode_version' is not equal to '$REQUIRED_XCODE_VERSION'"
-      printf "\033[0m"
-      exit 1
-    fi
-  fi
-}
+#!/usr/bin/env ruby
+
+return if ENV['DISABLE_XCODE_CHECK']
+
+installed_xcode_version = `xcodebuild -version`
+  .split("\n")
+  .first
+  .match(/Xcode ((\d+\.)?\d+\.\d+)/)[1]
+required_xcode_version = ENV['REQUIRED_XCODE_VERSION']
+
+if required_xcode_version.nil? || required_xcode_version.empty? 
```

Comment:
Move this block immediately after the https://github.com/guidomb/ios-scripts/pull/29/commits/8a4ad9fc22c008014f7c3a6b9025b0fd7fe4df88#diff-024cc101856d217cbc4835f5da6ee466R3 to avoid calling `xcodebuild` if `REQUIRED_XCODE_VERSION` is not available.